auth_manager.go

```diff
@@ -211,11 +211,23 @@ func (b *DefaultSessionManager) Sessions(filter string) []string {
 
 type DefaultKeyGenerator struct{}
 
+func generateToken(orgID, keyID string) string {
+	token, err := storage.GenerateToken(orgID, keyID, config.Global().HashKeyFunction)
+
+	if err != nil {
+		log.WithFields(logrus.Fields{
```

Comment:
we could also do `.WithError(err).WithFields` so error will appear as special error-field in log entry